LocalSettings.php

```diff
@@ -6821,7 +6821,7 @@
 		'zharkunuwiki' => "//$wmgUploadHostname/zharkunuwiki/4/41/Zharkunu_Logo_-_135.png",
 	),
 	'wgLogo' => array(
-		'default' => "//$wmgUploadHostname/metawiki/3/35/Miraheze_Logo.svg",
+		'default' => "//$wmgUploadHostname/metawiki/3/35/Miraheze_Logo.svg"
```

Comment:
needs to be 

		'default' => "//$wmgUploadHostname/metawiki/3/35/Miraheze_Logo.svg",

(there's a missing comma)